src/jit/emit.cpp

```diff
@@ -3341,23 +3341,23 @@ void emitter::emitDispIGlist(bool verbose)
 void emitter::emitDispGCinfo()
 {
     printf("Emitter GC tracking info:");
-    printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
+    // printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
     dumpConvertedVarSet(emitComp, emitPrevGCrefVars);
```

Comment:
I am planning to write dumpVarSet and use it there in the next PR.  A function with such name existed, I found two outdates comments that referenced here, but it was deleted before tfs history.